test/e2e/utils/utils.go

```diff
@@ -104,11 +105,15 @@ func UpdateResource(t *testing.T, obj UpdateObjects) (*unstructured.Unstructured
 	if err != nil {
 		t.Fatalf("Failed to create %s: %v", obj.GVR.Resource, err)
 	}
+	// Provide Sufficient time interval for messages to reach fack servers
+	time.Sleep(30 * time.Second)
```

Comment:
Adding sleep is not reliable and cleaner way.
We should have a retry logic with timeout to wait till the condition met

Reply:
trying out possibilities to understand the problem with GitHub actions, since there is no issues with local testing.

Reply:
Now it is assured that it is due to lack of time for messages reaching fack servers, will work on logics and update here